api/src/db.js

```diff
@@ -163,4 +164,37 @@ if (UNIT_TEST_ENV) {
 
     throw new Error(`Error while saving docs: ${errors.join(', ')}`);
   };
+
+  const DEFAULT_SECURITY_STRUCTURE = {
+    names: [],
+    roles: [],
+  };
+
+  module.exports.addRoleToSecurity = async (dbname, role, addAsAdmin) => {
```

Comment:
Taking `addAsAdmin` as a boolean makes it hard to read when invoking it... Consider adding two functions: `addRoleToAdmins` and `addRoleToMembers` to make it clear.